modules/nw-route-specific-annotations.adoc

```diff
@@ -66,6 +67,15 @@ This value is applicable to re-encrypt and edge routes only. For more informatio
 `if-none`: sets the header if it is not already set.| `ROUTER_SET_FORWARDED_HEADERS`
 
 |===
+[.small]
+--
+1. If the number of IP addresses and CIDR ranges in an allowlist exceeds 64, they are written into a separate file that is then referenced from `haproxy.config`.
++
+[NOTE]
+====
+To ensure that the addresses are written to the allowlist, check that the full list of CIDR ranges are listed in the Ingress Controller configuration file. The etcd object size limit restricts how large a route annotation can be. Because of this, it creates a threshold for the maximum number of IP addresses and CIDR ranges that you can include in an allowlist.
+====
+--
```

Comment:
my suggestion is to add this portion just after line 44 to have a continuation.

Reply:
The technical limitations of how tables are currently rendering makes content unreadable inside a cell, but I can take a look at a later time whenever the [issue](https://github.com/redhataccess/ascii_binder/issues/184) is resolved